### PR TITLE
`Pickup::PreviousOwner` support in CustomItems

### PR DESCRIPTION
**Pull request by IRacle1**
idk, maybe need other parameter in this like `needSetPreviousPlayer`, but i dont want to make breaking changes

### CODE REVIEW

File: Exiled.CustomItems/API/Features/CustomWeapon.cs [hunk 1]
```diff
@@ -93,7 +94,37 @@ public override Pickup Spawn(Vector3 position)
...
+                return null;
+            }
+
+            if (item is Firearm firearm && Attachments is not null && !Attachments.IsEmpty())
+                firearm.AddAttachment(Attachments);
+
+            Pickup pickup = item.Spawn(position);
+            if (pickup is null)
+            {
+                Log.Debug($"{nameof(Spawn)}: Pickup is null.");
```
Comment (NaoUnderscore) [inline Exiled.CustomItems/API/Features/CustomWeapon.cs:114]: Missing `Instance.Config.Debug`.

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

File: Exiled.CustomItems/API/Features/CustomItem.cs [hunk 2]
```diff
@@ -469,28 +468,60 @@ public static IEnumerable<CustomItem> UnregisterItems(IEnumerable<Type> targetTy
...
+        /// <returns>The <see cref="Pickup"/> of the spawned <see cref="CustomItem"/>.</returns>
+        public virtual Pickup Spawn(Player player, Item item, Player previousOwner = null) => Spawn(player.Position, item, previousOwner);
+
         /// <summary>
         /// Spawns a <see cref="Item"/> as a <see cref="CustomItem"/> where a specific <see cref="Player"/> is.
         /// </summary>
         /// <param name="player">The <see cref="Player"/> position where the <see cref="CustomItem"/> will be spawned.</param>
         /// <param name="item">The <see cref="Item"/> to be spawned as a <see cref="CustomItem"/>.</param>
         /// <returns>The <see cref="Pickup"/> wrapper of the spawned <see cref="CustomItem"/>.</returns>
-        public virtual Pickup Spawn(Player player, Item item) => Spawn(player.Position, item);
```
[the remark this answers is not in the document]
Comment (IRacle1) [reply Exiled.CustomItems/API/Features/CustomItem.cs:487]: Should I take part in the discussion? 
It's just that Joker made these changes, not me

[the remark this answers is not in the document]
Comment (IRacle1) [reply Exiled.CustomItems/API/Features/CustomItem.cs:487]: Also, I'm blind or you saying about another line?
image

[the remark this answers is not in the document]
Comment (NaoUnderscore) [reply Exiled.CustomItems/API/Features/CustomItem.cs:487]: Whatever.

**Review by Michal78900 — APPROVED** [review]

**Pull request merged**